vehicle/Sedan.h

```diff
@@ -5,8 +5,8 @@
 
 class Sedan final: public Car {
 public:
-    Sedan(IEngine* engine, IGear* gear)
-        : Car(engine, gear) {}
+    Sedan(IEngine* m_engine, IGear* m_gear)
```

Comment:
same here

Reply:
updated, p_ for parameters, m_ for class members